File: backend/Models/Payment.cs
namespace backend.Models
{
    public class Payment
    {
        public int Id { get; set; }
        public int StudentId { get; set; }
        public decimal Amount { get; set; }
        public DateTime PaymentDate { get; set; } = DateTime.UtcNow;
        public string Method { get; set; } = string.Empty; // Mobile Money, Bank Transfer
        public string ReceiptNumber { get; set; } = string.Empty;
        public string Status { get; set; } = "Completed"; // Pending, Completed, Failed
    }
}

File: frontend_backup/src/Types/Payment.ts
export interface Payment {
  id: number;
  studentId: number;
  amount: number;
  method: "Cash" | "Card" | "Online Transfer";
  referenceNumber: string;
  date: string;
  term: string;
  status: "Completed" | "Pending" | "Failed";
}
